test/data_aggregator.test.ts

```diff
@@ -180,25 +194,6 @@ describe('DataAggregator', () => {
           quoteCurrency: expectedQuoteCurrency,
         }
 
-        describe('when no adapters are specified in the config', () => {
```

Comment:
What happens in this case now?

Reply:
The oracle runs and doesn't publish anything.